docs/source/getting-started/docker-machine-drivers.adoc

```diff
@@ -101,13 +101,46 @@ $ sudo chown root:wheel $(brew --prefix)/opt/docker-machine-driver-xhyve/bin/doc
 $ sudo chmod u+s $(brew --prefix)/opt/docker-machine-driver-xhyve/bin/docker-machine-driver-xhyve
 ----
 
-For more information, see the GitHub documentation of the https://github.com/zchee/docker-machine-driver-xhyve#install[xhyve driver].
+For more information, see the GitHub documentation for the https://github.com/zchee/docker-machine-driver-xhyve#install[xhyve driver].
 
 [[manual-install]]
 === Manual install
 
+The `docker-machine-driver-xhyve` binary can be downloaded to any one of the following directories:
+
+-  The `/usr/local/bin/docker-machine-driver-xhyve` directory.
```

Comment:
/usr/local/bin/docker-machine-driver-xhyve is not a directory. /usr/local/bin is the directory